easybuild/framework/easyblock.py

```diff
@@ -246,6 +260,20 @@ def close_log(self):
         fancylogger.logToFile(self.logfile, enable=False)
 
     #
+    # INIT UTILITIES
+    #
+    def init_dry_run(self):
+        """Initialise easyblock instance for performing a dry run."""
+
+        # replace build/install dir
+        self.builddir = os.path.join(tempfile.gettempdir(), '__ROOT__', self.builddir.lstrip(os.path.sep))
+        dry_run_msg("Using fake build directory: %s" % self.builddir, silent=self.silent)
```

Comment:
In every dry run message being printed, the fake build/install dir is now replaced with the original build/install dir, so no more `__ROOT__` in the output, but the build/install dir is still replaced as was the case here to make sure only paths in the tmpdir are used/created/touched